lib/tokens/token.js

```diff
@@ -71,30 +77,20 @@ module.exports = function (log, random, P, hkdf, Bundle, error) {
         stack: err.stack
       })
     }
+    details.createdAt = optionallyOverrideCreatedAt(details.createdAt)
```

Comment:
IIRC I was thinking about doing something like this, which we now do as a result of a separate PR:

  https://github.com/mozilla/fxa-auth-server/blob/master/lib/routes/account.js#L2299